app/views/static/index.html.haml

```diff
@@ -1,4 +1,4 @@
-- set_meta_tags description: 'Get structured data out of the web. Code collaboration through GitHub. Run your scrapers in the cloud.'
+- set_meta_tags description: 'Hassle-free web scraping. Over 5000 free public scrapers, or write your own in Ruby, Python, PHP, Perl or Node. Push to the cloud and then set and forget.'
```

Comment:
Nice. Could we make the count dynamic? We're bound to forget to update this as the number changes.